be/src/vec/functions/function_regexp.cpp

```diff
@@ -501,6 +502,84 @@ struct RegexpExtractAllImpl {
     }
 };
 
+struct RegexpCountImpl {
+    static constexpr auto name = "regexp_count";
+
+    static void execute_impl(FunctionContext* context, ColumnPtr argument_columns[],
```

Comment:
maybe we only need impl template<bool const>execute_impl.
can remove the unless code func `execute_impl_const_args`